main.js

```diff
@@ -5,7 +5,7 @@ var express = require("express"),
 app.use(express.json())
 app.use(express.urlencoded({ extended: true }))
 
-var port = process.env.PORT || process.env.OPENSHIFT_NODEJS_PORT || 9090,
+var port = process.env.HDD_SUPERVISOR_PORT || process.env.OPENSHIFT_NODEJS_PORT || 9090,
```

Comment:
Now that we are renaming this, lets keep the name consistent rename process.env.OPENSHIFT_NODEJS_PORT also as process.env.HDD_SUPERVISOR_PORT

Reply:
Should I just omit process.env.OPENSHIFT_NODEJS_PORT ?

Reply:
aah yes you can omit 1
anyway both of them are from same source just 2 different env vars(from common parent source)..

Reply:
exactly . I made the changes in the morning itself but forgot to notify you.